scripts/ci/build-openssl.sh

```diff
@@ -57,7 +57,7 @@ if [ "$MACOS_UNIVERSAL_BUILD" == "true" ]; then
       -fPIC \
       --prefix=$build_folder \
       --openssldir=$build_folder \
-      no-tests no-shared "${@:2}"
+      no-tests no-shared enable-legacy enable-weak-ssl-ciphers "${@:2}"
```

Comment:
@notjaywu we will also need to enable these on the curl-for-windows fork, and then pull the latest on the git submodule - otherwise here they are only enabled for MacOS and Linux

Reply:
yes, this is another pr https://github.com/notjaywu/curl-for-windows/pull/1